components/bl-badge-component/src/content/image.js

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+
+export function Image({ imageUrl, imageWidth }) {
+  const styles = useMemo(() => ({
```

Comment:
as far as I know there is no reason to wrap style with useMemo if they go to a native DOM component